docs/openCypher.md

```diff
@@ -0,0 +1,62 @@
+# openCypher Data Format
+
+openCypher was chosen as it represents “the most widely adopted, fully-specified, and open query
+language for property graph databases”. Allowing customers to import data from enterprise applications
+such as neo4j and Amazon Neptune with greater ease. One caveat to this is that there are slight differences
+in the System column header formats used by each, these differences are shown below. For more info regarding 
+the data format see [here](https://docs.aws.amazon.com/neptune/latest/userguide/bulk-load-tutorial-format-opencypher.html)
+#### Entity (Node) System column headers
+- Neptune / neo4j
+- :ID     / _id  –  (Required) An ID for the node -> VERTEX (Gaffer)
+- :LABEL  / _labels  –  A label for the node -> GROUP (Gaffer)
+
+#### Edge (relationship) System column headers
+- Neptune   / neo4j
+- :ID       / _id   –   An ID for the relationship. There doesn't exist a direct mapping between "ID" in openCypher and 
+                  a dedicated attribute for an Edge with Gaffer, instead this is added as a property belonging to the Edge. 
+- :START_ID / _start  –   (Required) The node ID of the node this relationship starts from -> SOURCE (Gaffer)
+- :END_ID   / _end   –   (Required) The node ID of the node this relationship ends at -> DESTINATION (Gaffer)
+- :TYPE     / _type   –   A type for the relationship -> GROUP (Gaffer)
+
+#### Property column headers
```

Comment:
We should maybe document that `-` is replaced with `_`, and other non valid characters are stripped (and link to [Propertiesutil](https://github.com/gchq/Gaffer/blob/f16de7c3eccfe7a800cad1d7eea5fbae4cf01d44/core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/PropertiesUtil.java#L26))